### PR TITLE
Sort QuoteBundle based on typeOfContract

### PR DESCRIPTION
**Pull request by sampettersson**
# Jira Issue: [IPC-183]

## What?

> Sorting QuoteBundle based on `typeOfContract`

## Why?

> IPC needed help with this, and this is a feature request from App and Web clients.

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

[IPC-183]: https://hedvig.atlassian.net/browse/IPC-183

### CODE REVIEW

**Review by hedvigoscar — APPROVED** [review]
LGTM!

File: src/resolvers/cross-schema/quoteBundleOrderedQuotesExtension.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            transforms: [
+              {
+                transformRequest: AddFieldToDelegatedRequest(schemas(SchemaIdentifier.UNDERWRITER), "BundledQuote", "typeOfContract")
+              }
+            ]
+          }).then((quoteBundle: QuoteBundle) => {
+            const sortedBundle = {
+              ...quoteBundle, quotes: quoteBundle.quotes.sort((quoteA, quoteB) => {
+                if (quoteA.typeOfContract.includes('HOME_CONTENT')) return -1
+                if (quoteB.typeOfContract.includes('HOME_CONTENT')) return 1
```
Comment (meletios) [inline src/resolvers/cross-schema/quoteBundleOrderedQuotesExtension.ts:60]: There if statements have the exact same condition but different returns. Is it the thing that we want?

Comment (sampettersson) [reply src/resolvers/cross-schema/quoteBundleOrderedQuotesExtension.ts:60]: @meletios `quoteA` vs `quoteB` hehe

Comment (meletios) [reply src/resolvers/cross-schema/quoteBundleOrderedQuotesExtension.ts:60]: OH MY

Comment (meletios) [reply src/resolvers/cross-schema/quoteBundleOrderedQuotesExtension.ts:60]: I need glasses 👓

**Pull request merged**